sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -900,6 +900,46 @@ class SQLAppStatusListenerSuite extends SharedSparkSession with JsonTestUtils
       assert(innerMetric.isDefined)
     }
   }
+
+  test("SPARK-37578: Update output metrics from Datasource v2") {
+    withTempDir { dir =>
+      val statusStore = spark.sharedState.statusStore
+      val oldCount = statusStore.executionsList().size
```

Comment:
is this different from `statusStore.executionsCount()`?

Reply:
It is the same, I think.

Reply:
then `statusStore.executionsCount()` looks more efficient? It's also used in the code below.

Reply:
Yea, I will change to `executionsCount`.